packages/react/src/component-tests/IcPageHeader/IcPageHeader.cy.tsx

```diff
@@ -0,0 +1,282 @@
+    cy.checkA11yWithWait();
+    cy.scrollTo("bottom").checkHydrated("ic-button#end");
+    cy.checkA11yWithWait();
+    cy.get("ic-page-header").invoke("css", "position", "absolute");
+    cy.compareSnapshot({
+      name: "stickyVariant",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+    cy.get("ic-page-header").invoke("css", "position", "null");
+  });
+
+  it("page header should not stay on top if prop use stickyDesktopOnly for viewport extra small", () => {
+    mount(<DesktopScrollablePageVariant />);
+    cy.viewport("iphone-6");
+    cy.wait(500).compareSnapshot({
+      name: "desktopOnlyBeforeScroll",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+    cy.scrollTo(0, 800);
+    cy.wait(500).compareSnapshot({
+      name: "dasktopOnlyAfterScroll",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+  });
+
+  it("centered page header", () => {
+    mount(
+      <IcPageHeader
+        aligned="center"
+        heading="Latte recipe"
+        subheading="A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam. "
+        reverseOrder
+      >
+        <IcChip slot="heading-adornment" label="BETA" size="large" />
+        <IcButton slot="actions" variant="primary">
+          Create coffee
+        </IcButton>
+        <IcButton slot="actions" variant="tertiary">
+          Filter coffee
+        </IcButton>
+        <IcTextField
+          slot="input"
+          placeholder="Search for ingredients…"
+          label="Input"
+          hideLabel
+        />
+      </IcPageHeader>
+    );
+    cy.viewport(1450, 500);
+    cy.compareSnapshot({
+      name: "centerPageHeader",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("small size page header", () => {
+    mount(
+      <IcPageHeader
+        size="small"
+        heading="Latte recipe"
+        subheading="A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam."
+      />
+    );
+    cy.compareSnapshot({
+      name: "smallSize",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("fullWidth page header", () => {
+    mount(
+      <IcPageHeader
+        aligned="full-width"
+        heading="Latte recipe"
+        subheading="A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam. "
+        reverseOrder
+      >
+        <IcChip slot="heading-adornment" label="BETA" size="large" />
+        <IcButton slot="actions" variant="primary">
+          Create coffee
+        </IcButton>
+        <IcButton slot="actions" variant="tertiary">
+          Filter coffee
+        </IcButton>
+        <IcTextField
+          slot="input"
+          placeholder="Search for ingredients…"
+          label="Input"
+          hideLabel
+        />
+      </IcPageHeader>
+    );
+    cy.viewport(1440, 590);
```

Comment:
Where possible it's best to set the viewport before the mount, it's easier to review and can avoid breakpoint snaphots failing.